content/cloudflare-one/static/documentation/connections/authenticated-doh.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/env python3
+ 
+import os
+import subprocess
+import json
+import time
+from pprint import pprint
+ 
+verbose = os.environ.get('VERBOSE', False)
+ 
+ 
+def check_for_command(command, install):
+    try:
+        subprocess.check_output(["command", "-v", command])
+    except:
+        print(
+            f"Couldn't find {command} command. Please install with `{install}`")
+        exit(1)
+ 
+ 
+check_for_command("wg", "brew install wireguard-tools")
+check_for_command("curl", "brew install curl")
+ 
+ 
+def read_string_option(env_name, human_name):
+    v = os.environ.get(env_name, '')
+    if len(v) == 0:
+        v = input(f"Please input {human_name} > ")
+    if len(v) == 0:
+        print("Invalid {human_name}")
+        return read_string_option(env_name, human_name)
+    return v
+ 
+ 
+def request_create_user(user_name, user_email):
+    url = f"{CF_API}/{account_tag}/access/users"
+    # could also take a `custom` attribute with a map, e.g. {"custom": { "datacenter": "DFW" }}
+    body = {'name': user_name, 'email': user_email}
+    return request_post(url, body)
+ 
+ 
+def request_list_client_ids():
+    client_ids_response = request_get(
+        f"{CF_API}/{account_tag}/access/service_tokens/")
+ 
+    return " ".join(response['client_id'] for response in client_ids_response['result'])
+ 
+ 
+def request_create_service_token(service_token_name):
+    response = request_post(
+        f"{CF_API}/{account_tag}/access/service_tokens/", {'name': service_token_name})
+    client_id = response['result']['client_id']
+    client_secret = response['result']['client_secret']
+    return (client_id, client_secret)
+ 
+ 
+def request_list_service_tokens():
+    return request_get(f"{CF_API}/{account_tag}/access/service_tokens/")
+ 
+ 
+def request_enable_service_token_for_doh(service_token_id):
+    url = f"{CF_API}/{account_tag}/access/organizations/doh/{service_token_id}"
+    return request_post(url)
```

Comment:
missing a second param for the body, this will fail to call the function. Maybe:
```suggestion
    return request_post(url,"")
```